benchmark/C/Savina/src/include/matrix.h

```diff
@@ -76,24 +82,11 @@ double* mat_at_d(matrix_t matrix, size_t i, size_t j);
  * @return A pointer to the requested matrix entry, or NULL if 'matrix'
  *  is of the wrong type.
  */
-int* mat_at_i(matrix_t matrix, size_t i, size_t j);
-
-/*
- * Set the (i, j) entry of the given double matrix.
- * @param matrix the matrix to be modified.
- * @param i The row to be accessed.
- * @param j The column to be accessed.
- * @param value The value to be placed in the matrix.
- */
-void mat_set_d(matrix_t matrix, size_t i, size_t j, double value);
-
-/*
- * Set the (i, j) entry of the given integer matrix.
- * @param matrix the matrix to be modified.
- * @param i The row to be accessed.
- * @param j The column to be accessed.
- * @param value The value to be placed in the matrix.
- */
-void mat_set_i(matrix_t matrix, size_t i, size_t j, int value);
```

Comment:
Looks like set definitions were also removed.

Reply:
Oh! Thanks for the catch. I will amend that and inline the functions, and we can see if anything changes.

Reply:
Thanks! I did the same in #717.